examples/query_lang/credentials_alternatives.json

```diff
@@ -0,0 +1,43 @@
+{
+  "rules": {
+    "required": 2,
+    "from": [
+      { "ref": "pid" },
+      {
+        "required": 1,
+        "from": [
+          { "ref": "pid_reduced" },
+          { "ref": "residence_info" }
+        ]
+      }
+    ]
+  },
+  "credentials": {
```

Comment:
how would someone specify claims from different formats?
if that is allowed, we should have an example.
if not allowed, then it seems the `format` property can be extracted to a higher level.